backend/config/lightning.go

```diff
@@ -15,17 +15,33 @@
 
 package config
 
+import (
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/accounts/types"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/jsonp"
+)
+
+// LightningAccountConfig is the configuration of a single Lightning account.
+type LightningAccountConfig struct {
+	// Mnemonic is the wallet node generated from the device entropy.
+	Mnemonic string `json:"mnemonic"`
+	// RootFingerprint is fingerprint of the keystore that generated the entropy.
+	RootFingerprint jsonp.HexBytes `json:"rootFingerprint"`
```

Comment:
the account number could be added, but then again it could also be added later if we actually have multiple

Reply:
I was thinking about having it embedded in the code in the format `v0-<fingerprint>-ln-<account-number>`. Did you have something different in mind?

Reply:
i think the account code should never be parsed for information, it is simply an ID and should be treated like it's random. the account codes have already changed once (they were in fact random) in the past and they might change again, in which case worrying about all the places where the ID carries information would be hard.

anyway, maybe it's good to add the AccountNumber field (set to 0) already now to avoid the temptation of parsing the account code in the future :joy:

Reply:
Yeah that makes sense :) Done